modules/govuk/manifests/apps/rummager.pp

```diff
@@ -67,5 +98,15 @@
     "${title}-PUBLISHING_API_BEARER_TOKEN":
       varname => 'PUBLISHING_API_BEARER_TOKEN',
       value   => $publishing_api_bearer_token;
+    'AMQP_EXCHANGE':
```

Comment:
I think the queue and exchange name belong in the code repo - they feel like equivalent level "configuration" to a table name in a database.  The queue and exchange won't vary depending on deployment decisions (unlike the hosts, user and password settings).

That is the pattern followed by other consumers, such as the panopticon one.